config/default.yml

```diff
@@ -531,6 +531,11 @@ Layout/SpaceInsideStringInterpolation:
     - space
     - no_space
 
+Layout/Tab:
+  # By default, the indentation width from Layout/IndentationWidth is used
+  # But it can be overridden by setting this parameter
+  IndentationWidth: ~
```

Comment:
I think it's more common that a tab represents 8 or 4 steps of indentation, and that people/editors mix tabs and spaces. Or do people use tabs representing to positions, just because that's the indentation width?

Reply:
That is a good question. My team has tab set to 2 spaces to handle indentation for us. 

I think in most cases, if a tab is used, it will be used for indentation. In that case, we would want to replace the tab character with spaces based on the indentation width. 

The other place it could be used is to separate code. This should be a single space or column aligned. In either case, I think we have cops to handle that spacing.

The tests for `Layout/Tab` appear to be testing tab as indentation.

Reply:
And I see now that `Tab#autocorrect` replaces tab with two spaces on the current `master`, so go with the default that you chose. It's backwards compatible.